cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/PredicateFunctionsTest.scala

```diff
@@ -81,42 +81,42 @@ class PredicateFunctionsTest extends DocumentingTest {
       }
     }
     section("exists()", "functions-exists") {
-      p("Returns true if a match for the pattern exists in the graph, or the property exists in the node, relationship or map.")
-      function("exists(pattern-or-property)", ("pattern-or-property", "A pattern or a property (in the form 'variable.prop')."))
+      p("`exists()` returns true if a match for a given pattern exists in the graph, or if a specified property exists in the node, relationship or map.")
+      function("exists(pattern-or-property)", "A Boolean.", ("pattern-or-property", "A pattern or a property (in the form 'variable.prop')."))
       query(
         """MATCH (n)
           |WHERE exists(n.name)
           |RETURN n.name AS name, exists((n)-[:MARRIED]->()) AS is_married""".stripMargin, ResultAssertions((r) => {
           r.toList should equal(List(Map("name" -> "Alice", "is_married" -> false), Map("name" -> "Bob", "is_married" -> true), Map("name" -> "Charlie", "is_married" -> false), Map("name" -> "Daniel", "is_married" -> false), Map("name" -> "Eskil", "is_married" -> false)))
         })) {
-        p("This query returns the names of all nodes with a name property along with a boolean `true` / `false` indicating if they are married.")
+        p("The names of all nodes with a `name` property is returned, along with a boolean `true` / `false` indicating if they are married.")
```

Comment:
`"... property is returned, along with a ..."` --> `"... property ARE returned, along with a ..."`?

Reply:
👍